README.md

```diff
@@ -1,10 +1,10 @@
 # Public Curation Protocol
 
 ### High-level overview
-1. CurationFactory.sol allows for easy creation of individual curation contracts
```

Comment:
I may be useful to state the difference between curation contracts and individual curation contracts because it's not all the way clear

Reply:
in reality u deploy a proxy of Curator.sol when you deploy something from the factory. need to abstract this some way so we dont use the word "proxy"